lib/widgets/movie/cast.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:flutter/material.dart';
+import 'package:movie_card/widgets/movie/cast_card.dart';
+import 'package:movie_card/shared/ui_consts.dart';
+
+class Cast extends StatelessWidget {
```

Comment:
This is a good addition but remember that if you add the cast now, on the future when you call the real api you need to also get the cast from there